recipes/xr_fresh/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "xr_fresh" %}
+{% set version = "v0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mmann1123/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 130acba402bf1357ad9439c7835d5e9cd090d011bbb5d57a9d159e868dfa8d14
+
+build:
+  skip: true  # [py<37 or py>311]
+  skip: true  # [win]
+ 
+  script: |
+    {{ PYTHON }} -m pip install . -vv
+    {% if win %}
+    {{ PYTHON }} -m pip install "jax[cpu]" -f https://storage.googleapis.com/jax-releases/jax_releases.html -vv
```

Comment:
This is not OK, you should not be vendoring any other packages. Please skip on Windows for now until we have jaxlib there.

Reply:
removing windows jax install